tutorials/content/procedural_geometry.rst

```diff
@@ -3,8 +3,8 @@
 Procedural geometry generation
 ==============================
 
-Users often ask how to generate geometry from easily code. This is not very complicated, but it's not obvious.
-Godot provides a few classes entirely dedicated to make it this easy. Still, the best tool for the job depends
+Users often ask how to generate geometry easily from code. This is not very complicated, but it's not obvious.
+Godot provides few classes entirely dedicated to make it this easy. Still, the best tool for the job depends
```

Comment:
You should not have removed the "a", Without it this sentence is broken. It should read "Godot provides a few classes" as it did previously.

Reply:
i will fix it.